config-connector/solutions/iam/kpt/storage-bucket-iam/README.md

```diff
@@ -0,0 +1,44 @@
+Storage Bucket IAM
+==================================================
+# NAME
+  storage-bucket-iam
+# SYNOPSIS
+  Config Connector compatible yaml to enable permissions for a storage bucket.
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/config-connector/solutions/iam/kpt/storage-bucket-iam storage-bucket-iam
+  ```
+# REQUIREMENTS
+- A working Config Connector installation.
+- The cnrm-system service account must have `roles/storage.admin`.
+- A storage bucket managed by [IAM](https://cloud.google.com/storage/docs/access-control).
+
+  Note: Using uniform bucket-level access control is recommended for this package.
```

Comment:
As we discussed, you'll also want to add a link for "uniform bucket-level access control" to give some context to the customers.

Reply:
Added link